### PR TITLE
Rename: `find_gips()` -> `find_MAP()`, `likelihood` -> `posteriori`

### PR DESCRIPTION
**Pull request by PrzeChoj**
Change names of functions:
1. `find_gips()` -> `find_MAP()`
2. `log_likelihood_of_gips()` -> `log_posteriori_of_gips()`
3. `log_likelihood_of_perm()` -> `log_posteriori_of_perm()`

Change names of variables:
1. `likelihood` -> `posteriori`
2. `g_opt` -> `g_map`

### CODE REVIEW

**Comment by PrzeChoj** [comment]
Waiting for approval @MrDomani

**Review by MrDomani — APPROVED** [review]
LGTM

**Pull request merged**
